hr_attendance_rfid/models/hr_employee.py

```diff
@@ -66,7 +66,5 @@ def register_attendance(self, card_code):
                 res['error_message'] = msg
                 return res
         except Exception as e:
-            msg = e.message
-            _logger.error(msg)
```

Comment:
I assume you are removing this for having tests correct, but what you can do it's a `mute_logger` in the test for that purpose.

Reply:
Aren't UserError exceptions running automatically in logs?

Reply:
No, the Travis status is determined if there's an `ERROR` text parsed in the log.